config.json

```diff
@@ -2135,24 +2135,24 @@
         "groupId": "org.reactivestreams",
         "artifactId": "reactive-streams",
         "version": "1.0.4",
-        "nugetVersion": "1.0.4.5",
+        "nugetVersion": "1.0.4.6",
         "nugetId": "Xamarin.Android.ReactiveStreams",
         "dependencyOnly": false,
         "templateSet": "reactive-streams"
       },
       {
         "groupId": "androidx.navigation",
         "artifactId": "navigation-common-ktx",
-        "version": "2.5.3",
-        "nugetVersion": "2.5.3.1",
+        "version": "2.6.0",
+        "nugetVersion": "2.6.0",
         "nugetId": "Xamarin.AndroidX.Navigation.Common.Ktx",
         "dependencyOnly": true
       },
       {
         "groupId": "androidx.navigation",
         "artifactId": "navigation-runtime-ktx",
-        "version": "2.5.3",
-        "nugetVersion": "2.5.3.1",
+        "version": "2.6.0",
```

Comment:
Same.

Reply:
uff 
and I was digging through my memory - what did I do wrong???